pkg/cmd/infra/router/template.go

```diff
@@ -205,6 +207,14 @@ func (o *TemplateRouterOptions) Complete() error {
 		return fmt.Errorf("invalid reload interval: %v - must be a positive duration", nsecs)
 	}
 
+	if o.ReloadGap < 0 {
```

Comment:
This should be in validate

Reply:
Should the other checks in here be in validate too?  Or is there some pattern that I don't grok fully.